### DISCUSSION TITLE
Kesinlikle ;

### DISCUSSION THREAD

**Discussion by ccksoner** [IBAX-io/go-ibax#1283] [Ideas]
Herkes, IBAX'in BaaS platformunu (Weaver) özelleştirilebilir bir konsensüs mekanizmaları ekosistemi oluşturmak ve basit sürükle ve bırak ile tam madeni para basımı oluşturmak için kullanabilir, bu da düşük kod geliştirme ve sıfır kod madeni para basımını mümkün kılar, bu da DApp'leri geliştirmenin süresini ve engelini büyük ölçüde azaltır . IBAX, Ethereum'dan çok daha hızlıdır, Avalanche'dan daha kolaydır ve Solana'dan daha merkezi değildir.

**Comment by ahmederoni** [comment]
Türkçe başlıklar arasında rastladıgım mantıklı ilk konu bu heralde,nereye baksan kaç para olacak kaç usd olacak gibi şeyler yazılmış şahsen kendi milletimden utanıyorum :(
